components/app-core/frontend/src/view/navbar/navigation/bottom-nav.html

```diff
@@ -0,0 +1,17 @@
+<div>
+  <div class="bottom-nav-copyright" translate>navbar.copyright</div>
+  <div class="bottom-nav-menu" id="navbar-user-menu-dropdown">
+    <div class="bottom-nav-menu-wrapper">
+      <div class="bottom-nav-menu dropup">
+        <div class="bottom-nav-menu-icon dropdown-toggle" data-toggle="dropdown">
+          <i class="material-icons">more_vert</i>
```

Comment:
The icon isn't vertically centred and appears unaligned with the copy write message or, when screen is super squished, the username.